pkg/rancher-components/src/components/Pill/RcTag/RcTag.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { RcTagProps } from './types';
+
+const props = withDefaults(defineProps<RcTagProps>(), { disabled: false, showClose: false });
+const emit = defineEmits(['close']);
+</script>
+
+<template>
+  <div
+    class="rc-tag"
+    :class="{[props.type]: true, disabled: props.disabled}"
+  >
+    <slot name="default" />
+    <button
+      v-if="props.showClose"
+      :aria-label="props.closeAriaLabel"
```

Comment:
I'm considering if we should provide a default value for `aria-label` since this is an icon button. We can provide some context that's better than an icon with something simple like `Close Tag` and developers can override this to provide more context about the specific tag.